lms/djangoapps/grades/models.py

```diff
@@ -27,7 +27,7 @@
 
 # Used to serialize information about a block at the time it was used in
 # grade calculation.
-BlockRecord = namedtuple('BlockRecord', ['locator', 'weight', 'max_score'])
+BlockRecord = namedtuple('BlockRecord', ['locator', 'weight', 'r_possible', 'graded'])
```

Comment:
ok.  I'm going to go ahead and fully spell out all the "r_" to "raw_" and "w_" to "weighted_", per this comment and @jcdyer's below.